### DISCUSSION TITLE
Option to configure next/image to fetch with proxy https_proxy

### DISCUSSION THREAD

**Discussion by barbara-lea** [vercel/next.js#21874] [Ideas]
**Describe the feature you'd like to request**

My feature request is very similar to the one mentioned [here](https://github.com/vercel/next.js/issues/20837). The NextJS application is deployed behind a firewall that requires the network calls to go through a proxy. This way the requests that the next/image does from the server side are failing. 

**Describe the solution you'd like**

I would like to be able to configure the [fetchOptions](https://github.com/vercel/next.js/pull/21400) that are used by next/image to fetch the images, or the fetchOptions' `agent` property. 

**Describe alternatives you've considered**

Setting the global proxy agent is not an option for me because that way other requests are failing that are done on server side in `getServerSideProps` functions.

**Comment by HeroadZ** [comment]
Is there any progress? I'm facing the same issue
@timneutkens

**Comment by amesas** [comment]
Same problem here. We are behind a corporate firewall and we have to use a custom server just to setup the Global Agent package.

Being able to setup fetchOptions, to setup our proxy would allow us to run on NextJS without custom server.

#21400 or #20837.

Is it really completed as marked in #21872?

Anyone knows if it's really possible to do it?

**Comment by amesas** [comment]
@timneutkens this one is important for corporate firewalls.

Next 13 is not using HTTPS_PROXY env everywhere, so we still have to user custom server

**Comment by Gr3at** [comment]
any updates on this one?
I am really surprised it's not addressed in a straightforward way.
does it affect so small % of dev or is there a solution that we are unaware of?

**Comment by Charismara** [comment]
Is there any progress on this?
